### PR TITLE
feat: t7 - get only the filenames

### PR DESCRIPTION
**Pull request by krasninja**
## Task

Script should get a folder as an argument and return a list of all unique files with extensions but without the absolute path (including in the nested directories). only `-f` option is allowed for the use of find command, i.e.: `find /tmp -type f`. Which should find all files in all nested dirs and then process the output by using expansion.

- File name: `./chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh`
- Task: https://github.com/saritasa-nest/saritasa-devops-camp/blob/main/chapter2%20-%20bash%20basics/l2/bash%20expansion.md#t7---get-only-the-filenames

## Usage

```
$ ./chapter2-l2-t7-get-unique-files.sh ~/1
2021-06-04_ДР.zip
4WheelThunder.cdi
Arkanoid (1987)(Imagine Software).tap
Battle City (VS) [p1][!].nes
Battle_City_(J).nes
Batty (1987)(Hit-Pak)[t].tap
Bomb Jack (1986)(Elite Systems).tap
Carmageddon Max Pack (1998).zip
Chip 'n Dale Rescue Rangers 2 (U) [!].nes
Contra (U) [!].nes
Contra Force (U) [!].nes
DOOM II (1994).zip
```

### CODE REVIEW

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (kseniyashaydurova) [inline chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh:1]: Please fix the same moments as in https://github.com/krasninja/devops-camp-2023/pull/6 PR

**Review by kseniyashaydurova — CHANGES_REQUESTED** [review]

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh (hunk 1 above)
Comment (krasninja) [reply chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh:1]: 1. Add dir input arg check.
2. Show errors of `find` command
3. Use global "dir".
4. Use `sort -u`.

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#   Writes file to stdout.
+#######################################
+function get_files {
+  for file in $(find "${dir}" -type f); do
+    echo "${file##*/}"
+  done
+}
+
+get_files | sort -u
+
```
Comment (kseniyashaydurova) [inline chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh]: Too many empty lines, should be just 1

**Review by kseniyashaydurova — APPROVED** [review]

File: chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh (hunk 2 above)
Comment (krasninja) [reply chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh]: Fixed.

**Pull request merged**